packages/react/components/modal/Modal.tsx

```
@@ -113,10 +82,44 @@ const Modal = ({
+        refBtnModal.current && refBtnModal.current.focus()
+        setIndexFocusable(0)
+      }
+      return active || false
+    })
+  }, [active])
+
+  useEffect(() => {
+    if (refModal.current) {
+      const footer = refModal.current.querySelector('[data-modal-footer]')
+      const getCTA = footer?.querySelectorAll('button')
+      if (getCTA) getCTA.forEach((el, i) => (refsActions.current[i + 1] = el))
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)